tests/unit/requirements.txt

```diff
@@ -0,0 +1 @@
+flake8
```

Comment:
I do not think we need `flake8`  in unit tests.

Reply:
You are right, it is not needed